scripts/content/delete.js

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env node
+
+// Dependencies
+const shell = require('shelljs'); // eslint-disable-line import/no-extraneous-dependencies
+const config = require('../../config'); // eslint-disable-line import/no-unresolved
+
+Object.keys(config.demo).forEach(producer => {
```

Comment:
shouldn't the logic be integrated within the cli instead of in a custom script? just wondering

Reply:
The logic is indeed in the CLI's delete command. This script is simply iterating through all producers and runs the CLI's logic for each of them.

Reply:
I mean, instead of learning a new command (`yarn content:delete`), wouldn't it be easier to just run `npx eubfr-cli delete --all`?

Reply:
The `--all` flag is currently scoped to a producer, if `--producer` is not provided, it defaults to agri. I could default it to all producers?